### PR TITLE
Feature/wmt 11/t2a detail report

### PR DESCRIPTION
**Pull request by Sam-Rodgers**

### CODE REVIEW

File: test/unit/services/test-get-export-csv.js [hunk 1]
```diff
@@ -78,6 +78,10 @@ describe('services/get-export-csv', function () {
       expect(getExportCsv(orgUnit.TEAM.name, helper.SUSPENDED_LIFER_EXPORT_RESULT.suspendedLiferExportDetails, tabs.EXPORT.SUSPENDED_LIFERS_EXPORT).csv).to.eql(helper.SUSPENDED_LIFER_CSV.csv)
     })
 
+    it('for ARMS Export', function () {
```
Comment (kevinfox1) [inline test/unit/services/test-get-export-csv.js:81]: Can ARMS be changed to T2A please? Thanks

Comment (Sam-Rodgers) [reply test/unit/services/test-get-export-csv.js]: That should be everything resolved now

File: app/services/get-export-csv.js [hunk 2]
```diff
@@ -40,6 +40,9 @@ const WORKLOAD_PERCENTAGE_BREAKDOWN_EXPORT_FIELDS = ['regionName', 'lduName', 't
 const SUSPENDED_LIFERS_EXPORT_FIELD_NAMES = ['Region Name', 'Probation Delivery Unit', 'Team Name', 'Tier Code', 'Row Type', 'CRN', 'Case Type', 'Offender Manager Name', 'Grade Code', 'In Custody?', 'Register Level', 'Register Category', 'Register Category Description', 'Registration Date']
 const SUSPENDED_LIFERS_EXPORT_FIELDS = ['regionName', 'lduName', 'teamName', 'tierCode', 'rowType', 'caseReferenceNo', 'caseType', 'offenderManagerName', 'gradeCode', 'inCustody', 'registerLevel', 'registerCategory', 'registerCategoryDescription', 'registrationDate']
 
+const T2A_EXPORT_FIELD_NAMES = ['Region Name', 'Region Id', ' lduName', 'lduId', 'TeamN ame', 'Team Id', 'CRN', 'Workload Owner Id', 'Om Name', 'Om Code', 'Event Number', 'Allocation Date', 'NSI Outcome Code', 'NSI Outcome Descript']
```
Comment (kevinfox1) [inline app/services/get-export-csv.js:43]: Could the following changes be made please:

- Remove the ID fields
- Change lduName to Probation Delivery Unit
- Change TeamN ame to Team Name
- Change Om Name to Offender Manager Name
- Change Om Code to Grade Code
- Change NSI Outcome Descript to NSI Outcome Description

Thanks

File: app/services/get-export-csv.js [hunk 3]
```diff
@@ -40,6 +40,9 @@ const WORKLOAD_PERCENTAGE_BREAKDOWN_EXPORT_FIELDS = ['regionName', 'lduName', 't
...
+const T2A_EXPORT_FIELDS = ['regionName', 'regionId', 'lduName', 'lduId', 'teamName', 'teamId', 'CRN', 'workload_owner_id', 'omName', 'omCode', 'Event_No', 'Allocation_Date', 'NSI_Outcome_Cd', 'NSI_Outcome_Desc']
```
Comment (kevinfox1) [inline app/services/get-export-csv.js:44]: Could the ID fields be removed please? Thanks

File: test/helpers/export-csv-helper.js [hunk 4]
```diff
@@ -40,6 +40,16 @@ module.exports.ARMS_EXPORT_CSV = {
     '"NPS North West","Cumbria","Rehabilitation","19-9-2018","CASEREF30065","A.N. Offender Manager N01B324NPSM","PO","Community","16-12-2016","16-12-2020"'
 }
 
+module.exports.T2A_EXPORT_CSV = {
```
Comment (kevinfox1) [inline test/helpers/export-csv-helper.js:43]: Could this test be updated to reflect the changes to the CSV header and the columns that have been removed? Thanks

**Review by kevinfox1 — CHANGES_REQUESTED** [review]

**Review by kevinfox1 — APPROVED** [review]
Looks good to merge

**Pull request merged**